src/components/cards/cards.css

```diff
@@ -0,0 +1,177 @@
+@import url('/src/components/utils.css');
+@import url('/src/components/badge/badge.css') ;
+
+.card{
+    font-family: 'Montserrat', sans-serif;
+    border-radius: 1rem;
+    box-shadow: 0px 1px 10px rgb(146 140 140 / 50%);
+}
+.img-responsive{
+    width : 100% ;
+    height : 100% ;
+    position : relative ;
+    }
+
+    
+.card-description{
+    display : flex ;
+    flex-direction: column;
+    justify-content : space-around ;
+    font-size : 1.5rem ;
+    margin-left : 1rem ;
+}
+.card-title{
+    font-weight : bold;
+}
+.card-description-primary .card-primary-text{
+    color : var(--ca-primary-color) ;
+    font-style: italic;
+}
+.card-description-primary .card-price{
+    color : var(--ca-secondary-color);
+}
+.star-badge .far{
+    color : var(--danger-text) ;
+    cursor: pointer;
+
+}
+.star-badge .far:hover{
+    font-size : 1.8rem ;
+    cursor: pointer;
+
+}
+.product-description{
+    font-size : 1.5rem ;
+}
+.card-cta{
+    display : flex ;
+    flex-wrap : wrap ;
+    justify-content: flex-start;
+    align-items : center ;
+    gap : 2rem ;
+}
+.fa-heart{
+    color : var(--danger-text) ;
+    font-size : 2.5rem ;
+    font-weight: 500;
+    cursor: pointer;
+}
+.fa-heart:hover{
+    font-weight : 600;
+}
+.btn-cta{
+    color : white ;
+    background-color: var(--ca-primary-color);
+    border-radius: 0.5rem ;
+    padding : 1.5rem ;
+    border : none ;
+    cursor: pointer;
+}
+.btn-cta:hover{
+    background-color : var(--ca-primary-color-lighter)
+}
+/* card : horizontal and vertical */
+
+.card-horizontal {
+    max-width : 60% ;
+    display: flex;
+    flex-direction: row;
+    justify-content: center ;
+    align-content: center;
+    width: 100%;
+    gap: 1.2rem;
+}
+.card-image-horizontal {
+    width: 50rem;
+    height: 28rem ;
+    border: 0;
+    padding: 1rem;
+    position: relative;
+}
+.description-wrap{
+    display : flex ;
+    flex-direction: row;
+    justify-content: space-between;
+}
+.card-dismiss{
+    margin-right : 20px;
+    font-weight: bolder;
+    font-size : 3rem;
+    cursor:pointer;
+    
+}
+.card-dismiss:hover{
+    color : var(--ca-primary-color)
+}
+.card-vertical {
+    max-width : 30% ;
+    height : auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center ;
+    align-content: center;
+    gap: 0.8rem;
+    padding : 1rem ;
+}
+.card-image-vertical{
+    width: 100%;
+    height: auto ;
+    border: 0;
+    padding: 1rem;
+    position: relative;
+}
+.card-vertical .card-description{
+    gap : 2.0rem ;
+    padding-top : 1rem ;
+}
+.card-badge{
+    position : absolute ;
+    top : 0 ;
+    left : 0 ;
+    font-size : 1.5rem ;
+    background-color : var(--success-text);
+    color : var(--white-color);
+    padding : 1rem ;
+    border-radius : 3rem ;
+}
+
+/* Card : text overlay and text only */
+
+.card-text-overlay{
+    width : 45% ;
+}
```

Comment:
This width 45% will create problems, you have to find a way to create this card keeping width: 100%.
Because in this way it would be very hard to use this card in ecomm